docs/src/process.rst

```diff
@@ -95,6 +97,13 @@ Data types
              * extensions like '.exe' or '.cmd'.
              */
             UV_PROCESS_WINDOWS_FILE_PATH_EXACT_NAME = (1 << 7)
+            /*
+             * Set the child process' supplementary group ids. The group ids are supplied
+             * in the 'gids' field in the options struct, and the number of groups is
+             * specified in the 'num_gids' field.  This does not work on windows;
+             * setting this flag will cause uv_spawn() to fail.
+             */
+            UV_PROCESS_SETGROUPS = (1 << 9)
```

Comment:
Can you please document UV_PROCESS_WINDOWS_USE_PARENT_ERROR_MODE above so there is no gap?